documentation/src/docs/table-api.md

```diff
@@ -156,17 +178,27 @@ pravega.tableSinkWriterBuilder()
         .forStream(stream)
         .withPravegaConfig(setupUtils.getPravegaConfig());
 
-StreamTableDescriptor desc = tableEnv.connect(pravega)
-        .withFormat(new Json().failOnMissingField(true).deriveSchema())
-        .withSchema(new Schema().field("category", Types.STRING()).field("value", Types.INT()))
-        .inAppendMode();
-desc.registerTableSink("test");
+ConnectTableDescriptor desc = tableEnv.connect(pravega)

```

Comment:
ConnectTableDescriptor is not needed here. `tableEnv.connect().xxx.createTemporaryTable()` is fine here.